legal-api/migrations/versions/01dc3337e726_add_batch_processing_status.py

```diff
@@ -0,0 +1,37 @@
+"""add_batch_processing_status
```

Comment:
It turned out to be a bit tricky to update the db enum for batch_processing status types using alembic. Since alembic [ignores the enum value change](https://freeman.vc/notes/automatically-migrate-enums-in-alembic), I had to update it using a [mostly SQL approach](https://stackoverflow.com/questions/14845203/altering-an-enum-field-using-alembic/16821396#16821396).